File: CS/XPServerModeViewDemo/Form1.cs
using DevExpress.Xpo;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using XpoTutorial;

namespace XPServerModeViewDemo {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        readonly Session session;

        public Form1() {
            InitializeComponent();
            session = new Session(XpoDefault.DataLayer);
            xpServerModeView1.ResolveSession += (s, e) => {
                e.Session = session;
            };
            xpInstantFeedbackView1.ResolveSession += (s, e) => {
                e.Session = session;
            };
        }
    }
}

File: VB/XPServerModeViewDemo/Form1.vb
Imports DevExpress.Xpo
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports XpoTutorial

Namespace XPServerModeViewDemo
	Partial Public Class Form1
		Inherits DevExpress.XtraEditors.XtraForm

		Private ReadOnly session As Session

		Public Sub New()
			InitializeComponent()
			session = New Session(XpoDefault.DataLayer)
			AddHandler xpServerModeView1.ResolveSession, Sub(s, e)
				e.Session = session
			End Sub
			AddHandler xpInstantFeedbackView1.ResolveSession, Sub(s, e)
				e.Session = session
			End Sub
		End Sub
	End Class
End Namespace
